pkgs/development/interpreters/php/default.nix

```diff
@@ -61,28 +63,37 @@ let
   , xmlrpcSupport ? (config.php.xmlrpc or false) && (libxml2Support)
   , cgotoSupport ? config.php.cgoto or false
   , valgrindSupport ? (config.php.valgrind or true) && (versionAtLeast version "7.2")
+  , ipv6Support ? config.php.ipv6 or true
+  , pearSupport ? (config.php.pear or true) && (libxml2Support)
   }:
 
     let
-      mysqlBuildInputs = optional (!mysqlndSupport) libmysqlclient;
       libmcrypt' = libmcrypt.override { disablePosixThreads = true; };
+      pear-nozlib = fetchurl {
+        url = "https://pear.php.net/install-pear-nozlib.phar";
```

Comment:
@etu this file is updated periodically. Is this use case suitable?

Reply:
@Izorkin That's why I think we should ditch the idea to not use the official tarballs because of this missing file.

That was a suggestion in hope that they would be the same. Then it showed, they wasn't the same. And I think we should have abandoned the idea to fetch from github there.

Like I mentioned in this comment: https://github.com/NixOS/nixpkgs/pull/73151#discussion_r347797884

Reply:
`https://pear.php.net/package/PEAR` - This package is rarely updated.

Reply:
@Izorkin Let's use the official tarballs instead of fetching the source from github.

Reply:
I do not know how to convert PEAR-1.10.10.tgz to install-pear-nozlib.phar

Reply:
You don't need to do that if we just revert 08bdfcf to fetch the official tarballs.

Reply:
No other ideas? I liked the fetchFromGitHub variant.

Reply:
Yeah, it would have been great if it worked. But since they seem to put things in the tarballs. And what they do to produce the tarballs may change over time. I don't think we should put effort into replicating this.

Reply:
Is it possible to moved it in a separate package? - php-pear?

Reply:
How is it possible to leave a temporary variant? If something changes, we will fix it later.

Reply:
Removed this variant.